DTH for WWST-4943, VISION Z-Wave Recessed D/W Sensor ZD 2105BR-5 - ANZ

ganeshasprabhu opened the pull request:
Added FP (finger print) for AU region. Please review and approve.

Added Below Line:
fingerprint mfr: "0109", prod: "2022", model: "2201", deviceJoinName: "Vision Recessed Door Sensor" //AU

Commit:
DTH for WWST-4943, VISION Z-Wave Recessed D/W Sensor ZD 2105BR-5 - ANZ

greens reviewed (changes requested):
Please match the spacing (tabs) of the rest of the file and rebase to fix conflicts.

ganeshasprabhu committed:
Merge branch 'master' into srib-wwst-VisionRecessedDoorSensor

greens commented on devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy:
```diff
@@ -53,10 +53,11 @@ metadata {
-		fingerprint mfr: "0371", prod: "0002", model: "0007", deviceJoinName: "Aeotec Door/Window Sensor 7" //US
-		fingerprint mfr: "0060", prod: "0002", model: "0003", deviceJoinName: "Everspring Door/Window Sensor" //US & EU
+		fingerprint mfr: "0371", prod: "0002", model: "0007", deviceJoinName: "Aeotec Door/Window Sensor 7" //US          
+    fingerprint mfr: "0060", prod: "0002", model: "0003", deviceJoinName: "Everspring Door/Window Sensor" //US & EU
```
You're still using spaces instead of tabs

ganeshasprabhu committed:
Tabs are incorporated.

I was using Linux vi editor, tab was set to 4 space. Corrected it now. Please review. Thank you.
